web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -16,11 +16,18 @@ interface ExchangeArrowProps {
   viewportWidth: number;
   viewportHeight: number;
   map: MapboxMap;
+  colorBlindMode: boolean;
 }
 
-function ExchangeArrow({ data, viewportWidth, viewportHeight, map }: ExchangeArrowProps) {
+function ExchangeArrow({
+  data,
+  viewportWidth,
+  viewportHeight,
+  map,
+  colorBlindMode,
+}: ExchangeArrowProps) {
   const mapZoom = map.getZoom();
-  const colorBlindModeEnabled = false; // TODO: FIX https://linear.app/electricitymaps/issue/ELE-1384/set-up-colorblind-mode-that-changes-co2-scale
+  const colorBlindModeEnabled = colorBlindMode; // TODO: FIX https://linear.app/electricitymaps/issue/ELE-1384/set-up-colorblind-mode-that-changes-co2-scale
```

Comment:
```suggestion
  const colorBlindModeEnabled = colorBlindMode;
```
I think we can remove this TODO now since the exchange part is fixed at least.

Reply:
Yes for sure! I just committed the suggestion. Thank you for through it!